crates/optimism/node/tests/e2e/p2p.rs

```diff
@@ -77,25 +83,9 @@ async fn can_sync() -> eyre::Result<()> {
         .wait_block(
             side_payload_chain[0].0.block().number,
             side_payload_chain[0].0.block().hash(),
-            true,
+            false,
```

Comment:
this marks whether we need to wait for `Finish` stage to advance which might not happen if persistence threshold is not reached